utils/releaser/tests/IntervalEvaluatorTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\Releaser\Tests;
+
+use Iterator;
+use PHPUnit\Framework\TestCase;
+use Shopsys\Releaser\IntervalEvaluator;
+
+final class IntervalEvaluatorTest extends TestCase
+{
+    /**
+     * @var \Shopsys\Releaser\IntervalEvaluator
+     */
+    private $intervalEvaluator;
+
+    protected function setUp(): void
+    {
+        $this->intervalEvaluator = new IntervalEvaluator();
+    }
+
+    /**
+     * @dataProvider provideData
```

Comment:
please can you give here link on custom docblock variables like this?

Reply:
it is not custom, we use it as well in our tests...https://phpunit.de/manual/3.7/en/appendixes.annotations.html#appendixes.annotations.dataProvider